index.html

```diff
@@ -13,7 +13,19 @@
 
 <body>
     <div class="container">
-   
+        <header class="header">
+            <a href="index.html"><img src="img/lambda-black.png" alt=""></a>
```

Comment:
Great use of the Lambda logo by wrapping it in `<a>` tags to redirect back to the home page.